src/common/redis_module/ray_redis_module.cc

```diff
@@ -832,8 +832,10 @@ int ResultTableLookup_RedisCommand(RedisModuleCtx *ctx,
 
   RedisModuleString *task_id;
   RedisModuleString *is_put;
+  RedisModuleString *data_size;
+  RedisModuleString *hash;
   RedisModule_HashGet(key, REDISMODULE_HASH_CFIELDS, "task", &task_id, "is_put",
-                      &is_put, NULL);
+                      &is_put,"data_size", &data_size, "hash", &hash,  NULL);
   RedisModule_CloseKey(key);
   if (task_id == NULL || is_put == NULL) {
```

Comment:
should we also check if `data_size` and `hash` are NULL here? is it possible for some of these to be NULL and the others to not be NULL?